src/main/webapp/WEB-INF/tags/instructor/feedbacks/feedbackSessionActions.tag

```diff
@@ -74,37 +74,13 @@
     data-placement="top"
     style="display: inline-block; padding-right: 5px;">
   <div class="btn-group margin-bottom-7px">
-    <a class="btn btn-default btn-xs btn-tm-actions session-remind-for-test<c:if test="${not actions.allowedToRemind || actions.privateSession}"> disabled</c:if>"
-        href="${actions.allowedToRemind && not actions.privateSession ? actions.remindLink : 'javascript:;'}"
-        data-fsname="${actions.fsName}">
+    <a class="btn btn-default btn-xs btn-tm-actions session-remind-particular-for-test<c:if test="${not actions.allowedToRemind || actions.privateSession}"> disabled</c:if>"
```

Comment:
Rather than an `a` without `href`, see if using `button` instead works.

Reply:
It works, but the following test function doesn't:
https://github.com/TEAMMATES/teammates/blob/497bf6da33843286a45bf99e3ec646c6a9c3ec2c/src/test/java/teammates/test/pageobjects/AppPage.java#L807-L813
Shall I create another `verifyUnclickableButton()` ? Or maybe overload the existing function like so: `public void verifyUnclickable(WebElement element, String element)`?

Reply:
@wkurniawan07 actually I would like to ask a related question as well, should we have more unit tests instead of integration test? In that case we can have more efficient testing. For Bao's question I think a new testing function will be better but correct me if I am wrong.

Reply:
@bqnguyen94 then you should make the button `disabled` using attribute instead of using class, no? Try and see if that works.

@LuLechuan if you're talking about JS unit test, yes I do agree, however our JS test suite is not really suitable for unit tests. I have been thinking of adopting new frameworks like Karma/Jasmine/..., but never got the time to actually work on it, especially now that I have a full-time day job.